run.go

```diff
@@ -261,26 +261,46 @@ func ScreenScale() float64 {
 	return uiDriver().ScreenScale()
 }
 
-// IsCursorVisible returns a boolean value indicating whether
-// the cursor is visible or not.
+// CursorMode returns the current cursor mode.
 //
-// IsCursorVisible always returns false on mobiles.
+// On browsers, only CursorModeVisible and CursorModeHidden are supported.
 //
-// IsCursorVisible is concurrent-safe.
-func IsCursorVisible() bool {
-	return uiDriver().IsCursorVisible()
+// CursorMode returns 0 on mobiles.
+//
+// CursorMode is concurrent-safe.
+func CursorMode() InputCursorMode {
+	return InputCursorMode(uiDriver().CursorMode())
 }
 
-// SetCursorVisible changes the state of cursor visiblity.
+// SetCursorMode sets the render and capture mode of the mouse cursor.
+// CursorModeVisible sets the cursor to always be visible.
+// CursorModeHidden hides the system cursor when over the window.
+// CursorModeCaptured hides the system cursor and locks it to the window.
+//
+// On browsers, only CursorModeVisible and CursorModeHidden are supported.
 //
-// SetCursorVisible does nothing on mobiles.
+// SetCursorMode does nothing on mobiles.
 //
-// SetCursorVisible is concurrent-safe.
+// SetCursorMode is concurrent-safe.
+func SetCursorMode(mode InputCursorMode) {
+	uiDriver().SetCursorMode(driver.CursorMode(mode))
+}
+
+// IsCursorVisible is deprecated as of 1.11.0-alpha.2. Use CursorMode instead.
```

Comment:
`1.10.0-alpha` is fine

Reply:
1.10.0-alpha or 1.11.0-alpha?

Reply:
Ah sorry, I meant `1.11.0-alpha`.

(I might remove all `-alpha` suffix from 'as of' later, but let's keep the consistency so far.)